webknossos/webknossos/cli/_utils.py

```diff
@@ -0,0 +1,334 @@
+"""Utilities to work with the CLI of webknossos."""
+
+import re
+from collections import namedtuple
+from enum import Enum
+from os import environ, sep
+from pathlib import Path
+from types import TracebackType
+from typing import Any, Generator, Iterator, Optional, Tuple, Type, Union
+
+import numpy as np
+from upath import UPath
+
+from webknossos import BoundingBox, Mag
+from webknossos.geometry.vec3_int import Vec3Int
+
+# constant for knossos dataset
+CUBE_EDGE_LEN = 128
+CUBE_SIZE = CUBE_EDGE_LEN**3
+CUBE_SHAPE = (CUBE_EDGE_LEN,) * 3
+KNOSSOS_CUBE_REGEX = re.compile(
+    rf"x(\d+){re.escape(sep)}y(\d+){re.escape(sep)}z(\d+){re.escape(sep)}(.*\.raw)$"
+)
+
+KnossosDatasetInfo = namedtuple("KnossosDatasetInfo", ("dataset_path", "dtype"))
+VoxelSize = namedtuple("VoxelSize", ("x", "y", "z"))
+Vec2Int = namedtuple("Vec2Int", ("x", "y"))
+
+
+class KnossosDataset:
```

Comment:
You could move all the Knossos-related code to convert_knossos or is it used in multiple modules?